docs/pages/reference/backends.mdx

```diff
@@ -111,8 +111,10 @@ and user records will be stored.
 To configure Teleport for using etcd as a storage backend:
 
 - Make sure you are using **etcd versions 3.3** or newer.
+- Follow [etcd cluster hardware recommendations](https://etcd.io/docs/v3.5/op-guide/hardware/) especially to leverage 
```

Comment:
I think this should be reworded. Something about "follow especially" doesn't sound quite right.

Reply:
@zmb3 please see latest update